lib/src/platform_specifics/android/styles/media_style_information.dart

```diff
@@ -0,0 +1,14 @@
+import 'default_style_information.dart';
+
+class MediaStyleInformation extends DefaultStyleInformation {
+
+  MediaStyleInformation({
+    bool htmlFormatContent = false,
+    bool htmlFormatTitle = false,
+  }) : super(htmlFormatContent, htmlFormatTitle);
+
+  Map<String, dynamic> toMap() {
```

Comment:
this method can be removed since it no longer has additional logic and just calls the base implementation

Reply:
I agree on the `toMap` method. Should I also remove the constructor?

Reply:
unless i'm mistaken, this class doesn't seem to be needed anymore even. if so, let's remove it